kickstarter/ios-oss
# Change Email Unverified Email state UX tweak

# 📲 What

Small UX improvement that prevents the user's email fields from being refreshed when the user's change email request is submitted. This means that they won't see the `This email is unverified` label until *after* they leave the screen and come back to it.

# 🤔 Why

We decided in combination with Design that this was a better user experience.

# ✅ Acceptance criteria

- [x] Test Case One Below:
1. Log in on an account with a verified email address. Navigate to change email screen. You *should not* see the "This email address is unverified" label or the "Send Verification Email" button. 
2. Change your email. You should see the "Got it, your changes have been saved" success banner. You *should not* see the "This email address is unverified" label or the "Send Verification Email" button. 
3. Navigate back to "Account". You should see the red warning icon in the "Change Email" row. 
4. Tap "Change Email" to navigate to the Change Email screen. You *should* see the "This email address is unverified" label *and* the "Send Verification Email" button.
- [x] Test Case Two Below:
1. Log in on an account with an *unverified* email address. Navigate to change email screen. You *should* see the "This email address is unverified" label and the "Send Verification Email" button. 
2. Change your email. You should see the "Got it, your changes have been saved" success banner. You *should still* see the "This email address is unverified" label and the "Send Verification Email" button.
